docs/source/containerization/docker.mdx

```diff
@@ -58,18 +58,32 @@ docker run -p 4000:4000 \
   --rm \
   --interactive \
   --tty \
-  --entrypoint=sh
-  ghcr.io/apollographql/router:<image version>-debug
-/dist # pwd
+  --entrypoint=bash \
+  ghcr.io/apollographql/router:<image version>
+root@0c6ae96a2bbd:/dist# pwd
```

Comment:
Do we need this prefix ?

Reply:
No. It's just to show what it would look like. I can take them out.

Reply:
done in: https://github.com/apollographql/router/pull/2142/commits/2bd3a4e0e4758a470e6351749f2fb3d2598bf376